### ISSUE TITLE
Accessibility: Improve accessibility of Jupyter FAQ page

### ISSUE THREAD

**Issue by Mackenzie-OO7** [jupyterhub/outreachy#71]
### What page is this for?

https://jupyter.org/governance/conduct/faq.html

### WAVE accessibility report

![Screenshot 2022-10-17 074917](https://user-images.githubusercontent.com/97461848/196107953-132a8ff5-37f9-4790-9e7b-ed34fb0c55f7.png)

### The HTML element to be changed

```html
<html style="margin-left: 380px;">

<label for="toctree-checkbox-1">
<i class="fas fa-chevron-down"></i>
</label>

<label for="toctree-checkbox-2">
<i class="fas fa-chevron-down"></i>
</label>

<label for="toctree-checkbox-3">
<i class="fas fa-chevron-down"></i>
</label>

<label for="__page-toc" class="headerbtn headerbtn-page-toc">
<span class="headerbtn__icon-container">
<i class="fas fa-list"></i>
</span>
</label>

<label for="__navigation" class="headerbtn" data-toggle="tooltip" data-placement="right" title="" data-original-title="Toggle navigation">
<span class="headerbtn__icon-container">
<i class="fas fa-bars"></i>
</span>
</label>

<input type="checkbox" class="sidebar-toggle" name="__navigation" id="__navigation" aria-label="Toggle navigation sidebar">

<input type="checkbox" class="sidebar-toggle" name="__page-toc" id="__page-toc" aria-label="Toggle in-page Table of Contents">

<button onclick="toggleFullScreen()" class="headerbtn" data-toggle="tooltip" data-placement="bottom" title="" data-original-title="Fullscreen mode">
<span class="headerbtn__icon-container">
<i class="fas fa-expand"></i>
</span>
</button>

<a class="reference external" href="mailto:conduct%40jupyter.org">
conduct
<span>
@
</span>

jupyter
<span>
.
</span>

org
</a>

<img src="https://licensebuttons.net/p/zero/1.0/88x31.png" style="border-style: none;" alt="CC0">

<footer class="footer-article noprint">
<!-- Previous / next buttons-->
<div class="prev-next-area">
<a class="left-prev" id="prev-link" href="reporting_events.html" title="previous page">
<i class="fas fa-angle-left"></i>
<div class="prev-next-info">
<p class="prev-next-subtitle">
previous
</p>
<p class="prev-next-title">
Reporting Guide - Jupyter Events
</p>
</div>
</a>
<a class="right-next" id="next-link" href="../governance.html" title="next page">
<div class="prev-next-info">
<p class="prev-next-subtitle">
next
</p>
<p class="prev-next-title">
Governance of Project Jupyter
</p>
</div>
<i class="fas fa-angle-right"></i>
</a>
</div>
</footer>
```

### Your proposed HTML (or CSS), after change

```html
//add the valid lang attribute value.
<html lang="en" style="margin-left: 380px;">

//Remove label elements for form controls not containing text, and hidden form controls.
<i class="fas fa-chevron-down"></i>

<i class="fas fa-chevron-down"></i>

<i class="fas fa-chevron-down"></i>

<span class="headerbtn headerbtn__icon-container">
<i class="fas fa-list"></i>
</span>

<span class="headerbtn headerbtn__icon-container" data-toggle="tooltip" data-placement="right" title="Toggle navigation">
<i class="fas fa-bars"></i>
</span>

//Change aria-label to aria-labelledby since multiple labels are necessary here. 
<input type="checkbox" class="sidebar-toggle" name="__navigation" id="__navigation" aria-labelledby="Toggle navigation sidebar">

<input type="checkbox" class="sidebar-toggle" name="__page-toc" id="__page-toc" aria-labelledby="Toggle in-page Table of Contents">

//Add appropriate text to the button
<button onclick="toggleFullScreen()" class="headerbtn" data-toggle="tooltip" data-placement="bottom" title="" data-original-title="Fullscreen mode"> Fullscreen Mode
<span class="headerbtn__icon-container">
<i class="fas fa-expand"></i>
</span>
</button>

//Combine redundant links into one link
<a class="reference external" href="mailto:conduct%40jupyter.org">
our conduct help service
</a>

//Add appropriate alternative text
<img src="https://licensebuttons.net/p/zero/1.0/88x31.png" style="border-style: none;" alt="Creative Commons public domain">

//Change tag from <footer> to <nav> for better semantic structure
<nav class="nav-article noprint">
<!-- Previous / next buttons-->
<div class="prev-next-area">
<a class="left-prev" id="prev-link" href="reporting_events.html" title="previous page">
<i class="fas fa-angle-left"></i>
<div class="prev-next-info">
<p class="prev-next-subtitle">
previous
</p>
<p class="prev-next-title">
Reporting Guide - Jupyter Events
</p>
</div>
</a>
<a class="right-next" id="next-link" href="../governance.html" title="next page">
<div class="prev-next-info">
<p class="prev-next-subtitle">
next
</p>
<p class="prev-next-title">
Governance of Project Jupyter
</p>
</div>
<i class="fas fa-angle-right"></i>
</a>
</div>
</nav>
```

### More info

Alternatively, for the multiple form labels, the label element can be removed instead, since it is hidden.

**Comment by minrk** [comment]
Thanks for the suggestions! I believe the `aria-labelledby` should get an _id_ of another element containing a label. As it's used here where the value should be the label itself, it should be `aria-label`.

For the redundant links - are there really redundant links? It is just one `a` tag?

**Comment by Mackenzie-OO7** [comment]
> Thanks for the suggestions! I believe the `aria-labelledby` should get an _id_ of another element containing a label. As it's used here where the value should be the label itself, it should be `aria-label`.
> 
> For the redundant links - are there really redundant links? It is just one `a` tag?

Thanks for the review! 
For the redundant links, both the text, and the link are the same link.

**Comment by sgibson91** [comment]
Thank you for your contribution! Since the contribution period is now over, we will close this issue.
